analyses/cnv-chrom-plot/util/bin-coverage.R

```diff
@@ -0,0 +1,142 @@
+# Functions for calling CN statuses of genome bins
+#
+# C. Savonen for ALSF - CCDL
+#
+# 2020
+
+bp_per_bin <- function(bin_ranges, status_ranges) {
+  # Given a binned genome ranges object and another GenomicRanges object, 
+  # Return the number of bp covered per bin.
+  #
+  # Args:
+  #   bin_ranges: A binned GenomicRanges made from tileGenome. 
+  #   status_ranges:A GenomicRanges object to calculate what percent coverage of
+  #   each bin. 
+  #
+  # Returns:
+  #  a data.frame with bins x number of bp 
+  
+  # Find the portions of each segment that overlap with each bin.
+  bin_overlaps <- GenomicRanges::pintersect(
+    IRanges::findOverlapPairs(
+      bin_ranges,
+      status_ranges
+    )
+  )
+  
+  # Which segs are a part of which bins?
```

Comment:
```suggestion
  # Which bins do the segs in `bin_overlaps` overlap with?
```

I think rewording this will help make this step a bit clearer.